src/main/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataDao.java

```diff
@@ -123,39 +128,43 @@ public CellMetadataDao(SolrCloudCollectionProxyFactory solrCloudCollectionProxyF
 
     // Given a type of metadata and an experiment accession, this method retrieves the value of that metadata for the
     // cells in an experiment
-    public Map<String, String> getMetadataValues(String experimentAccession,
-                                                 String metadataType) {
-        // We need to do this because we don't know if the metadata type is a factor or a characteristic
-        var fields = ImmutableMap.<SingleCellAnalyticsSchemaField, Collection<String>>of(
-                CHARACTERISTIC_NAME, ImmutableSet.of(metadataType),
-                FACTOR_NAME, ImmutableSet.of(metadataType));
+    public ImmutableMap<String, String> getMetadataValues(String experimentAccession,
+                                                            String metadataType) {
+        var METADATA_VALUE_TARGET_FIELD_NAME = "metadata_value";
```

Comment:
it would be better to make it a `static string` like `INFERRED_CELL_TYPE_SOLR_VALUE` and `SINGLE_CELL_IDENTIFIER_SOLR_VALUE`

Reply:
:+1:
Changed.